example/glue/configure_includes/configure_common.mk

```diff
@@ -24,5 +24,6 @@ CONFIGURE_ARGS += \
   --enable-OMR_OMRSIG \
   --enable-fvtest \
   --enable-OMR_GC_SEGREGATED_HEAP \
+  --enable-OMR_GC_MODRON_CONCURRENT_MARK \
```

Comment:
I have basic integration of ConcurrentGC with fvtest -- compiles, runs without fail all current gc tests, instantiating MM_ConcurrentGC instead of MM_ParallelGlobalGC whenever gencon is selected for GC policy. But current test cases are insufficient to drive tests to point of concurrent kickoff and only global GC occurring in gencon test runs is a system GC at end of test. 

We need more work (a model for generating high-volume heaps with mutable object graphs) in fvtest/gctest before they can exercise gencon in significant ways. Meantime Java is consuming current gencon (with MM_ConcurrentGC and MM_Scavenger), and current fvtest/gctest runs fine with concurrentMark enabled. 

Are you sure about this?